src/hp-client-lib.js

```diff
@@ -635,6 +641,18 @@
                     delete readRequestResolvers[requestId];
                 }
             }
+            else if (m.type == "hpsh_response") {
```

Comment:
Make this `hpsh_response`. It's a good idea to not expose the word "hpsh" to external libraries/protocols.